mkdocs/technical/plugins/hooks.md

```diff
@@ -713,8 +713,9 @@ Not all the hooks have been documented yet. ( [help needed! &#128568;](https://d
     | Activated (1)       | Intervene when a plugin is enabled                 |
     | Deactivated (2)     | Intervene when a plugin is disabled                |
     | Settings schema (3) | Override how the plugin's settings are retrieved   |
-    | Load settings (4)   | Override how the plugin's settings are loaded      |
-    | Save settings (5)   | Override how the plugin's settings are saved       |
+    | Settings model (4)  | Override how the plugin's settings are retrieved   |
```

Comment:
Sorry but I don't know well these two hook, do they have the same description?

Reply:
I think so, the `settings_model` returns the Pydantic model of the settings, instead `settings_schema` a JSON schema... but summoning @pieroit to get confirmation of it